js/samples/advanced-csharp-multiple-resources/ReadMe.md

```diff
@@ -6,7 +6,7 @@
 
 ## Usage
 
-Microsoft 1st Party Office applications should use the `v0.5.6` Immersive Reader JavaScript SDK and the SDK's authentication token is retrieved by providing a `SubscriptionKey` and `Region`. This will ensure the Immersive Reader uses Office compliant APIs deployed to OSI.
+Microsoft 1st Party Office applications should use the `v0.5.7` Immersive Reader JavaScript SDK and the SDK's authentication token is retrieved by providing a `SubscriptionKey` and `Region`. This will ensure the Immersive Reader uses Office compliant APIs deployed to OSI.
```

Comment:
Bit of a run-on sentence. Should revise to:

Microsoft 1st Party Office applications should use the `v0.5.7` Immersive Reader JavaScript SDK. [Then the rest about the token... But first....]

...need to ask... The reason we're unifying this is so that we have 1 single internal SDK for both Client and S2S customers, correct? And when we ask ALL of those customers to start using this new SDK, then intent is for them to transition away from OSI, and instead start calling our Cognitive (LaaS/KLaaS) backend, right? And to do that, they will need to use AAD auth token.

If that is the case then we need to revise the remainder of this section accordingly to discuss authentication (AAD token for Client, Subscription Key for S2S) and remove references to OSI.

And if we are changing the auth requirements, then we also need to update a bit more, samples, reader URLs, etc.

Maybe this larger update is for a future 'final' unified version? And this version is just incremental merge. When we get to the more official transition version, we should increment the version number higher than 0.5

Reply:
resolving as this incremental and i will make a task to track the update of docs and samples when we plan to release the next internal SDK